backslash/lazy_query.py

```diff
@@ -8,13 +8,16 @@
 
 class LazyQuery(object):
 
-    def __init__(self, client, path=None, url=None, page_size=100):
+    def __init__(self, client, path=None, url=None, query_params=None, page_size=100):
         super(LazyQuery, self).__init__()
         self._client = client
         if url is None:
             url = client.api.url
         if path is not None:
             url = url.add_path(path)
+        if query_params is not None:
+            for (param, value) in query_params.iteritems():
```

Comment:
K. so iterating with query_params.keys() is the right way? (I'm getting mixed answers online)